app/components/article/model.js

```diff
@@ -70,7 +72,7 @@ function generateFeedImage(data) {
   }
 }
 
-module.exports.save = function (uri, data, locals) {
+module.exports.save = function(uri, data, locals) {
```

Comment:
I believe we should revert this rule and keep it with a space after `function` 
```suggestion
module.exports.save = function (uri, data, locals) {
```